docker-compose.yaml

```diff
@@ -61,6 +61,9 @@ services:
     expose:
       # ZMQ socket port.
       - 5555
+    volumes:
+      # Persistent volume to store sealed data across restarts
+      - ./.sealed:/project/avalon/.sealed
```

Comment:
Any reason we want to use "dot" in volume name and keep it hidden ?

Reply:
Just obfuscating as manual intervention or deleting the files therein might lead to an irrecoverable situation.